charts/values.yaml

```diff
@@ -43,6 +43,11 @@ django:
     db_user: ""
     db_host: ""
     db_port: ""
+    email_backend: ""
```

Comment:
Since this is explicitly set in `settings.py`, no need for a variable here.  Being explicit in `settings.py` means the environment can't override it.